docs/src/includes/_types.md

```diff
@@ -1930,8 +1940,10 @@ vector&lt;[JournalPage](#JournalPage)&gt; | [pages](https://github.com/spelunky-
 
 Type | Name | Description
 ---- | ---- | -----------
+int | [platform_icon](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=platform_icon) | 16 = PC, 17 = Discord, 18 = Steam, 19 = Xbox, 32 = [Switch](#Switch), 48 = PS, 49 = PS again?
```

Comment:
Switch, what a fuky name you have there, it would be funny if it already registered name somewhere else right